hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionFactory.java

```diff
@@ -47,6 +49,16 @@
  * }
  * </pre>
  *
+ * Since 2.2.0, Connection created by ConnectionFactory can contain user-specified kerberos
```

Comment:
This doc is misleading. `2.2.0` is not released yet. Is this referring to 2.1.0?